src/graphql/typeDefs/dataTypes/communityInput.graphql

```diff
@@ -0,0 +1,48 @@
+"Community Query input, use id OR slug, NOT BOTH"
+input CommunityQueryInput {
+  id: ID
+  slug: Slug
+}
+
+input CommunityCreateInput {
+  "Community's name"
+  name: String!
+
+  "unique community slug."
+  slug: Slug!
+
+  "Community Short Description"
+  description: String!
+
+  "A detailed description of Community"
+  aboutUs: String
+
+  "This communities logo"
+  logo: URL
```

Comment:
I feel like this should be required too.

Reply:
done